src/Kernel.php

```diff
@@ -28,7 +28,9 @@ class Kernel extends ZikulaKernel
 
     public function registerBundles(): iterable
     {
-        $bundleHelper = new PersistedBundleHelper($_ENV['DATABASE_URL'] ?? '');
+        $databaseUrl = $this->booted && $this->getContainer()->hasParameter('database_url') ? $this->getContainer()->getParameter('database_url') : '';
+        $bundleHelper = new PersistedBundleHelper($databaseUrl);
+//        $bundleHelper = new PersistedBundleHelper($_ENV['DATABASE_URL'] ?? '');
         $bundles = require $this->getProjectDir() . '/config/bundles.php';
```

Comment:
@Guite this is an idea but I've not really tested it. it requires the parameter change in services.yaml too. I'm not sure its better practice then using the GLOBAL. :-) wonder what you think.

Reply:
actually, I'm not even sure it WORKS. I'll be interested to hear your thoughts.

Reply:
Yes. Using a parameter seems the way to go :+1:

Reply:
doesn't work.